lib/Sema/CSDiag.cpp

```diff
@@ -3487,9 +3487,30 @@ typeCheckArgumentChildIndependently(Expr *argExpr, Type argType,
 /// Diagnose an argument labeling issue, returning true if we successfully
 /// diagnosed the issue.
 static bool diagnoseArgumentLabelError(Expr *expr,
```

Comment:
Looking closer, the place where it's called with null might have a better candidate.